manager/system_handler.py

```diff
@@ -157,10 +157,15 @@ def handle_get(cls, **kwargs):
 
         cves_view = SystemCvesView(list_arguments, {"rh_account_number": connexion.context['user'],
                                                     'inventory_id': inventory_id}, args, args, connexion.request.path)
+
+        opted_out = SystemPlatform.get(SystemPlatform.inventory_id == inventory_id).opt_out
```

Comment:
you can use `system.opt_out` instead as the fetch of the data is already done on the line 148.
```
            system = (SystemPlatform.select(
                SystemPlatform.last_evaluation, SystemPlatform.opt_out)
                      .where(SystemPlatform.inventory_id == inventory_id)
                      .where(SystemPlatform.rh_account == connexion.context['user'])
                      .get())
```